src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -2212,6 +2213,7 @@ class PartitionBoundSpecSegment(BaseSegment):
     As per https://www.postgresql.org/docs/13/sql-altertable.html.
     """
 
+    type = "partition_bound_segment"
```

Comment:
Could we name it this instead? Having a segment whose type mentions `segment` seems redundant. `partition_bound_spec` is the term used in the docs linked above.
```suggestion
    type = "partition_bound_spec"
```

Reply:
Done.